test_runner/fixtures/broker.py

```diff
@@ -31,7 +31,7 @@ def try_start(self):
             return
 
         listen_addr = self.listen_addr()
-        log.info(f'starting storage_broker to listen incoming connections at "{listen_addr}"')
+        log.info("%s", 'starting storage_broker to listen incoming connections at "{listen_addr}"')
```

Comment:
It seems we should use an f-string for the second argument here

Reply:
good catch, ty!